googleapis/bigtable/admin/v2/bigtable_instance_admin.pb.go

```diff
@@ -991,6 +991,132 @@ func (x *UpdateClusterMetadata) GetFinishTime() *timestamppb.Timestamp {
 	return nil
 }
 
+// The metadata for the Operation returned by PartialUpdateCluster.
+type PartialUpdateClusterMetadata struct {
```

Comment:
please do not merge - waiting to merge in https://github.com/googleapis/google-cloud-go/pull/5149

Reply:
Patched out. Note, this will need to be manually patched back in later. We should make sure this is tracked somewhere.

Reply:
I created https://github.com/googleapis/google-cloud-go/issues/5151. Thanks for fixing this!